test/unit/features/transition/transition-group.spec.js

```diff
@@ -293,5 +293,22 @@ if (!isIE9) {
       }).$mount()
       expect('<transition-group> children must be keyed: <div>').toHaveBeenWarned()
     })
+
+    it('component tag in transition group', () => {
+      const component = { template: '<div><slot></slot></div>' }
+      Vue.component('component', component)
```

Comment:
That is a problem. `<component>` is resolved in `<transition-group>`, if I register it in `App` it won't be available in `<transition-group>`.

> Created a feature request #5252